oopsla2024-benchmarks/Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM rust:1.74-bookworm
+RUN echo 'deb http://deb.debian.org/debian bookworm-backports main' >> /etc/apt/sources.list
+RUN wget https://packages.microsoft.com/config/debian/12/packages-microsoft-prod.deb -O packages-microsoft-prod.deb && \
+    dpkg -i packages-microsoft-prod.deb && \
+    rm packages-microsoft-prod.deb
+RUN apt-get update && apt-get install -y git python3-matplotlib python3-pandas golang-1.21-go dotnet-sdk-6.0 openjdk-17-jdk vim && \
```

Comment:
We don't need dotnet (anymore), do we? No Dafny.

Reply:
Correct